src/main/java/graphql/util/querygenerator/QueryGenerator.java

```diff
@@ -0,0 +1,147 @@
+package graphql.util.querygenerator;
+
+import graphql.ExperimentalApi;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLFieldsContainer;
+import graphql.schema.GraphQLInterfaceType;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLOutputType;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.GraphQLUnionType;
+
+import javax.annotation.Nullable;
```

Comment:
Small one: could you change this to be the JSpecify Nullable annotation, rather than javax?

I'll amend the ArchUnit test so this gets automatically picked up in future

Reply:
Done ;-)

Out of curiosity: what's the advantage of the JSpecify version of Nullable when compared to the native one?

Reply:
Yep good question, it is only an annotation at the end of the day so there isn't inherently something "new" about JSpecify's "implementation", but the advantage is that it has become a standard because all the big industry players are behind it. For example, Spring have formally adopted it, even at work JSpecify (& NullAway) will soon become a standard.